src/orders/components/OrderListDatagrid/datagrid.ts

```diff
@@ -0,0 +1,217 @@
+import {
+  loadingCell,
+  moneyCell,
+  readonlyTextCell,
+  tagsCell,
+  textCell,
+} from "@dashboard/components/Datagrid/customCells/cells";
+import { GetCellContentOpts } from "@dashboard/components/Datagrid/Datagrid";
+import { AvailableColumn } from "@dashboard/components/Datagrid/types";
+import { Locale } from "@dashboard/components/Locale";
+import { getMoney } from "@dashboard/components/Money/utils";
+import { OrderListQuery } from "@dashboard/graphql";
+import useLocale from "@dashboard/hooks/useLocale";
+import {
+  getStatusColor,
+  transformOrderStatus,
+  transformPaymentStatus,
+} from "@dashboard/misc";
+import { OrderListUrlSortField } from "@dashboard/orders/urls";
+import { RelayToFlat, Sort } from "@dashboard/types";
+import { getColumnSortDirectionIcon } from "@dashboard/utils/columns/getColumnSortDirectionIcon";
+import { GridCell, Item } from "@glideapps/glide-data-grid";
+import {
+  DefaultTheme,
+  ThemeTokensValues,
+  useTheme,
+} from "@saleor/macaw-ui/next";
+import moment from "moment-timezone";
+import { IntlShape, useIntl } from "react-intl";
+
+import { columnsMessages } from "./messages";
+
+export const getColumns = (
+  intl: IntlShape,
+  sort: Sort<OrderListUrlSortField>,
+  emptyColumn: AvailableColumn,
+): AvailableColumn[] => [
+  emptyColumn,
+  {
+    id: "number",
+    title: intl.formatMessage(columnsMessages.number),
+    width: 100,
+    icon: getColumnSortDirectionIcon(sort, OrderListUrlSortField.number),
+  },
+  {
+    id: "date",
+    title: intl.formatMessage(columnsMessages.date),
+    width: 200,
+    icon: getColumnSortDirectionIcon(sort, OrderListUrlSortField.date),
+  },
+  {
+    id: "customer",
+    title: intl.formatMessage(columnsMessages.customer),
+    width: 200,
+    icon: getColumnSortDirectionIcon(sort, OrderListUrlSortField.customer),
+  },
+  {
+    id: "payment",
+    title: intl.formatMessage(columnsMessages.payment),
+    width: 200,
+    icon: getColumnSortDirectionIcon(sort, OrderListUrlSortField.payment),
+  },
+  {
+    id: "status",
+    title: intl.formatMessage(columnsMessages.status),
+    width: 200,
+    icon: getColumnSortDirectionIcon(sort, OrderListUrlSortField.fulfillment),
+  },
+  {
+    id: "total",
+    title: intl.formatMessage(columnsMessages.total),
+    width: 150,
+  },
+];
+
+interface GetCellContentProps {
+  columns: AvailableColumn[];
+  orders: RelayToFlat<OrderListQuery["orders"]>;
+  loading: boolean;
+}
+
+export const useGetCellContent = ({
+  columns,
+  orders,
+  loading,
+}: GetCellContentProps) => {
+  const intl = useIntl();
+  const { locale } = useLocale();
+  const { theme: currentTheme, themeValues } = useTheme();
+
+  return (
+    [column, row]: Item,
+    { added, removed }: GetCellContentOpts,
+  ): GridCell => {
+    if ([-1, 0].includes(column)) {
```

Comment:
**issue**: I have no idea what that means, wrapping this by function, named with its intention would be helpful

Reply:
eg. `isFirstColumn(column: number) => [-1, 0].includes(column)`